### ISSUE TITLE
Is the interaction records sequentially ordered? 

### ISSUE THREAD

**Issue by jasongong11** [MengtingWan/goodreads#5]
Thank you for developing and sharing this great dataset. I am currently trying to create a sequence model for reading records. However, I am not sure if each user's interaction records are sequentially ordered. 

I noticed that, for each user, the book_id is monotonically decreasing. Does it imply that the records are sequentially ordered in a reverse manner? That the user rates the last row of records first, then rates the second last row of records, and rates the first row at the end. 

Thank you for your time!

**Comment by jasongong11** [comment]
For those who are interested. I explored the full (by genre) dataset and matched its rows with the simplified CSV dataset. 

Here is what I found:
1. The full dataset is sequentially ordered by the date added. But the order is **reversed**, meaning the earliest comes first, and the oldest comes at last row. 
2. The simplified dataset is also sequentially ordered. Its order is **chronological** (**_not reversed_**), meaning the oldest comes first, and the earliest comes at the last row.
